tests/keras_contrib/optimizers/lars_test.py

```diff
@@ -0,0 +1,6 @@
+from __future__ import print_function
+from keras_contrib.tests import optimizers
+from keras_contrib.optimizers import lars
+
+optimizers._test_optimizer(lars.LARS(0.01))
```

Comment:
Please put it in functions called `test_lars` and `test_lars_nesterov`.

Can you also add a `test_lars_skip` where you create a model and use `skip_list`? The model doesn't need to be big. a model.fit on a single epoch should be enough.